src/charm.py

```diff
@@ -33,6 +34,14 @@
 from tempo_config import TEMPO_ROLES_CONFIG
 
 logger = logging.getLogger(__name__)
+PEERS_RELATION_ENDPOINT_NAME = "peers"
+
+
+class PeerData(DatabagModel):
+    """Peer data between the coordinator units."""
+
+    hostname: str
```

Comment:
I think we should rename this to "fqdn", so it's more obvious what it means and more future-proof. What if the next time we need to add another type of hostname? what do we call it?